cms/djangoapps/contentstore/rest_api/v1/views/certificates.py

```diff
@@ -0,0 +1,106 @@
+""" API Views for course certificates """
+
+import edx_api_doc_tools as apidocs
+from opaque_keys.edx.keys import CourseKey
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from cms.djangoapps.contentstore.utils import get_certificates_context
+from cms.djangoapps.contentstore.rest_api.v1.serializers import (
+    CourseCertificatesSerializer,
+)
+from common.djangoapps.student.auth import has_studio_write_access
+from openedx.core.lib.api.view_utils import (
+    DeveloperErrorViewMixin,
+    verify_course_exists,
+    view_auth_classes,
+)
+from xmodule.modulestore.django import modulestore
+
+
+@view_auth_classes(is_authenticated=True)
+class CourseCertificatesView(DeveloperErrorViewMixin, APIView):
+    """
+    View for course certificate page.
+    """
+
+    @apidocs.schema(
+        parameters=[
+            apidocs.string_parameter(
+                "course_id", apidocs.ParameterLocation.PATH, description="Course ID"
+            ),
+        ],
+        responses={
+            200: CourseCertificatesSerializer,
+            401: "The requester is not authenticated.",
+            403: "The requester cannot access the specified course.",
+            404: "The requested course does not exist.",
+        },
+    )
+    @verify_course_exists()
+    def get(self, request: Request, course_id: str):
+        """
+        Get an object containing course's certificates.
+
+        **Example Request**
+
+            GET /api/contentstore/v1/certificates/{course_id}
+
+        **Response Values**
+
+        If the request is successful, an HTTP 200 "OK" response is returned.
+
+        The HTTP 200 response contains a single dict that contains keys that
+        are the course's certificates.
+
+        **Example Response**
+
+        ```json
+        {
+            "certificate_activation_handler_url": "/certificates/activation/course-v1:org+101+101/",
+            "certificate_web_view_url": "///certificates/course/course-v1:org+101+101?preview=honor",
```

Comment:
Is this url supposed to start with three forward slashes?

Reply:
Original method for this field is `get_lms_link_for_certificate_web_view` [here](https://github.com/openedx/edx-platform/blob/02dc68221b65a86fe7f6623e9eb161003e07cfc7/cms/djangoapps/contentstore/utils.py#L216) which contains `return "//{certificate_web_base}/certificates/course/{course_id}?preview={mode}".format()`. Therefore I can assume that this is the real value for the `certificate_web_view_url` field.